### PR TITLE
Beginning of API endpoints and user registration.

### PR DESCRIPTION
**Pull request by iheanyi**
Getting this show on the road, less talk, more code. Here's a feature for easy user registration, I got the ball rolling, who is going to follow my momentum?

### CODE REVIEW

**Comment by iheanyi** [comment]
@GRardB @reggie Am I gonna get this Code Review or nah? >_>

File: accounts/tests.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from unittest import skip
+
+class AccountsTest(APITestCase):
+    def test_create_user(self):
+        """
+        Ensure we can create a new user. Later on, ensure that the user model
+        also has an UserProfile model.
+        """
+        url = reverse('account-create')
+        data = { 'username': 'foobar', 'email': 'foobarbaz@example.com',
```
Comment (GRardB) [inline accounts/tests.py]: Is there any reason to not format this like:

    data = {
        'herp': 'derp',
        'foo': 'bar'
    }

**Comment by GRardB** [comment]
Ship it.

File: accounts/tests.py (hunk 1 above)
Comment (iheanyi) [inline accounts/tests.py]: For future coding, I can do this, lol.

**Pull request merged**